action.yaml

```diff
@@ -136,7 +136,7 @@ runs:
       if: ${{ inputs.provision == 'true' }}
       id: derive-image-name
       shell: bash
-      run: echo "image-name=$(echo ${{ inputs.image }} | sed 's/\-ci//g')_$(echo ${{ inputs.image-version }} | sed 's/\(.*\)\-\(.*\)/\1/g')" >> $GITHUB_OUTPUT
+      run: echo "image-name=$(echo ${{ inputs.image }} | sed 's/\-ci//g')_$(echo ${{ inputs.image-version }} | sed 's/\([^-]*\)\-\(.*\)/\1/g')" >> $GITHUB_OUTPUT
```

Comment:
Can you provide an example?

Reply:
Sure. Prior to this change, `kind:6.6-20240707.080256-amd64` would resolve to `kind_6.6-20240707.080256.qcow2` instead of `kind_6.6.qcow2` as expected. This essentially means that all image names with two `-` characters are broken in this way.

Reply:
Updated commit message to reflect this